server/src/routes/winner.ts

```diff
@@ -0,0 +1,15 @@
+import express from "express";
```

Comment:
Great start adding this in! This backend is actually part of a different task so please delete it or uncommit it.